docs/source-pytorch/data/alternatives.rst

```diff
@@ -20,13 +21,24 @@ However, Lightning also supports other data types such as plain list of batches,
     trainer = Trainer()
     trainer.fit(model, data)
 
-Examples for custom iterables include `NVIDIA DALI <https://github.com/NVIDIA/DALI>`__ or `FFCV <https://github.com/libffcv/ffcv>`__ for computer vision.
-Both libraries offer support for custom data loading and preprocessing (also hardware accelerated) and can be used with Lightning.
+Below we showcase Lightning examples with packages that compete with the generic PyTorch DataLoader and might be
+faster depending on your use case. They might require custom data serialization, loading, and preprocessing that
+is often hardware accelerated.
+
+.. TODO(carmocca)
```

Comment:
This is a comment: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#comments

Reply:
No, this is a nice comment!